### PR TITLE
Updating Tracker Docs for 4.0

### PR DESCRIPTION
**Pull request by russorat**
I still have to update the screenshots, but figured i'd start taking some review comments.

### CODE REVIEW

File: cdap-docs/tracker-manual/source/http-restful-api.rst [hunk 1]
```diff
@@ -809,3 +809,380 @@ Results (reformatted for display)::
...
+   * - ``200 OK``
+     - Returns the results in the body of the response
+   * - ``500 SERVER ERROR``
+     - Unknown server error
+
+Retrieve the Data Dictionary for the Schema
+-------------------------------------------
+Returns the data dictionary related to the specified schema::
+
+  POST /v3/namespaces/<namespace-id>/apps/_Tracker/services/TrackerService/methods/v1/dictionary
```
Comment (awholegunch) [inline cdap-docs/tracker-manual/source/http-restful-api.rst]: This is the same as the previous endpoint. Did you forget a parameter?

**Comment by russorat** [comment]
@awholegunch your changes LGTM. I'll upload the new images as soon as that code is checked in.

**Comment by awholegunch** [comment]
Running as a Quick Build: http://builds.cask.co/browse/CDAP-DQB201-1

Fails because image is missing.

Pages of interest: http://builds.cask.co/artifact/CDAP-DQB201/shared/build-1/Docs-HTML/4.0.0-SNAPSHOT/en/tracker-manual/index.html

**Comment by awholegunch** [comment]
Running as a Quick Build: http://builds.cask.co/browse/CDAP-DQB201-2 (passed!)

Pages of interest: http://builds.cask.co/artifact/CDAP-DQB201/shared/build-2/Docs-HTML/4.0.0-SNAPSHOT/en/tracker-manual/index.html

**Comment by awholegunch** [comment]
Running as a Quick Build: http://builds.cask.co/browse/CDAP-DQB201-3 (fails, but unrelated to this PR)

Fixed some incorrect links for the Tracker home page.

Pages of interest: http://builds.cask.co/artifact/CDAP-DQB201/shared/build-3/Docs-HTML/4.0.0-SNAPSHOT/en/tracker-manual/index.html

**Comment by russorat** [comment]
Lgtm

**Pull request merged**